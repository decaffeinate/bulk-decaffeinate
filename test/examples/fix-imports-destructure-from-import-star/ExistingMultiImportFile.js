import ConvertedCoffeeFile, {secondExport as s} from './ConvertedCoffeeFile';

console.log(ConvertedCoffeeFile.firstExport);
console.log(s);
