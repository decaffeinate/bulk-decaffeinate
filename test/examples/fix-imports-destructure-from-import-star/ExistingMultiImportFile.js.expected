import * as ConvertedCoffeeFile from './ConvertedCoffeeFile';

const {
  secondExport: s
} = ConvertedCoffeeFile;

console.log(ConvertedCoffeeFile.firstExport);
console.log(s);
